### PR TITLE
emit event eifupdated

### PR DESCRIPTION
**Pull request by arnav-deep**

### CODE REVIEW

File: contracts/enclaves/MarketV1.sol [hunk 1]
```diff
@@ -188,6 +190,15 @@ contract MarketV1 is
         _updateToken(_token);
     }
 
+    function _eifUpdate(bytes32 _job, string memory _metadata) internal {
```
Comment (roshanrags) [inline contracts/enclaves/MarketV1.sol]: rename to `_jobMetadataUpdate`

File: contracts/enclaves/MarketV1.sol [hunk 2]
```diff
@@ -188,6 +190,15 @@ contract MarketV1 is
...
+        jobs[_job].metadata = _metadata;
+        emit EifUpdated(_job, _metadata);
+    }
+
+    function eifUpdate(bytes32 _job, string calldata _metadata) external onlyJobOwner(_job) {
```
Comment (roshanrags) [inline contracts/enclaves/MarketV1.sol]: rename to `jobMetadataUpdate`

File: contracts/enclaves/MarketV1.sol [hunk 3]
```diff
@@ -165,6 +165,8 @@ contract MarketV1 is
 
     event TokenUpdated(IERC20 indexed oldToken, IERC20 indexed newToken);
 
+    event EifUpdated(bytes32 indexed job, string metadata);
```
Comment (roshanrags) [inline contracts/enclaves/MarketV1.sol]: rename to `JobMetadataUpdated`

**Review by roshanrags — CHANGES_REQUESTED** [review]

File: contracts/enclaves/MarketV1.sol (hunk 2 above)
Comment (arnav-deep) [reply contracts/enclaves/MarketV1.sol]: Done

File: contracts/enclaves/MarketV1.sol (hunk 1 above)
Comment (arnav-deep) [reply contracts/enclaves/MarketV1.sol]: Done

File: contracts/enclaves/MarketV1.sol (hunk 3 above)
Comment (arnav-deep) [reply contracts/enclaves/MarketV1.sol]: Done

File: contracts/enclaves/MarketV1.sol [hunk 4]
```diff
@@ -188,6 +196,15 @@ contract MarketV1 is
         _updateToken(_token);
     }
 
+    function _jobMetadataUpdate(bytes32 _job, string memory _metadata) internal {
```
Comment (roshanrags) [inline contracts/enclaves/MarketV1.sol]: put this along with the other _job*

File: contracts/enclaves/MarketV1.sol [hunk 5]
```diff
@@ -188,6 +196,15 @@ contract MarketV1 is
...
+        jobs[_job].metadata = _metadata;
+        emit JobMetadataUpdated(_job, _metadata);
+    }
+
+    function jobMetadataUpdate(bytes32 _job, string calldata _metadata) external onlyJobOwner(_job) {
```
Comment (roshanrags) [inline contracts/enclaves/MarketV1.sol]: put this along with the other job*

**Review by roshanrags — CHANGES_REQUESTED** [review]

**Review by roshanrags — APPROVED** [review]

**Pull request merged**